proto/qdrant/cloud/account/v1/account.proto

```diff
@@ -183,6 +235,13 @@ service AccountService {
     option (qdrant.cloud.common.v1.supported_actor_types) = ACTOR_TYPE_USER;
     // gRPC Gateway REST call
     option (google.api.http) = {delete: "/api/account/v1/accounts/{account_id}/members/{user_id}"};
+    // events
+    option (qdrant.cloud.event.v1.event_options) = {
+      event_type: EVENT_TYPE_DELETED
+      resource_type: "account-member"
```

Comment:
should we name the resource types with the same name as the proto message (e.g. `AccountMember`) instead of `account-member`? That could help in case we want to automate this behavior or provide default values. Same in case we want to add linting rules.